stores/identity.ts

```diff
@@ -89,7 +88,7 @@ export const useIdentityStore = defineStore('identity', {
         '1': '0',
         '5': '0',
       },
-      address: localStorage.getItem('kodaauth') || '',
```

Comment:
After releasing to prod env or canary env, the current login address would be automatically logged out. Would you like to be compatible with this case?

Reply:
done